### DISCUSSION TITLE
Where to find Fluent UI web component (@fluentui/web-components) reference wiki/docs?

### DISCUSSION THREAD

**Discussion by Bertie2011** [microsoft/fluentui#31925] [Q&A] [answered]
I know the web component docs are at: https://learn.microsoft.com/en-us/fluent-ui/web-components 
And that it's based on FAST: https://www.fast.design/docs/components

But because the Fluent UI web-component docs are incredibly minimal, I also used to peek at the editor auto completion files (*.vscode.definition.json) at https://github.com/microsoft/fluentui/tree/master/packages/web-components/src to understand which attributes and possible values are available.

For example, I haven't got a clue now which "appearance" values are allowed for text fields, since both docs at the top don't mention this and the source I used to reference no longer exists. (I do know from memory it is "filled" and "outline" at the very least, but you get my point). Also the React docs don't really match the web component implementation.

Does anybody know where I can find the desired documentation for my use case(s)?

**Comment by davatron5000** [comment, marked as answer]
@Bertie2011 Beyond the repo itself, we have a web component Storybook up that currently reflects the work in progress on v3 beta. 

https://web-components.fluentui.dev/?path=/docs/concepts-introduction--page

**Reply by Bertie2011** [reply]
Yeah there is still a mismatch between the v2 web components that I use and the v3 story books (understandably)

> Beyond the repo itself

I guess I could just use the git tags to find the definition files I was referring to earlier:
https://github.com/microsoft/fluentui/tree/%40fluentui/web-components_v2.6.1/packages/web-components/src
